capstone/src/main/java/com/step902020/capstone/Review.java

```diff
@@ -14,32 +21,99 @@
 
   long timestamp;
 
-  @Field(name="name")
-  String name;
+  int likes;
+
+  String individualName;
```

Comment:
Have you considered just using a reference to `Individual`?

Reply:
Yes that's what I had originally, but if you leave a review on an event that you've also saved, you get an infinite recursion error

Reply:
I see. This must be happening with JSON serialization. I think we need to come up with a general strategy for dealing with cycles.
Have you looked at `@JsonIdentityInfo`. That might solve the problem with cycles.